apps/iOS/navigation-bar.md

```diff
@@ -0,0 +1,7 @@
+## Navigation Bar
+
+The user navigates the app through a bottom navigation bar. The navigation bar contains icons representing various destinations within the app.
```

Comment:
rather than being broad here, we could say the exact navigation labels. So in future navigation change can be tracked as well.